File: AppGR/FontEnd/app/src/main/java/com/example/appgrit/models/UserRenewalModel.java
package com.example.appgrit.models;

public class UserRenewalModel {
    private String UserId;
    private float Price;

    // Constructors, getters, and setters

    public UserRenewalModel() {
    }

    public UserRenewalModel(String userId, float price) {
        UserId = userId;
        Price = price;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public double getPrice() {
        return Price;
    }

    public void setPrice(float price) {
        Price = price;
    }
}

File: AppGR/BackEnd/AppGrIT/Models/UserRenewalModel.cs
using System.ComponentModel.DataAnnotations;

namespace AppGrIT.Models
{
    public class UserRenewalModel
    {
        [Required]
        public string UserId
        { get; set; } = null!;

        [Required]
        public float Price
        { get; set; }
    }
}
